react/react1/week1/componentstables/src/Components/Thead.js

```diff
@@ -0,0 +1,12 @@
+function Thead(props) {
```

Comment:
You are not using any prop in this component. You should remove 'prop' from here.
```suggestion
function Thead() {
```